packages/db-client/src/client.ts

```diff
@@ -29,6 +37,19 @@ export class DbClient {
     this.closed = false;
   }
 
+  public async availabilityCaches(): Promise<
+    CollectionWrap<AvailabilityCache>
+  > {
+    const db = await this.db;
+    if (!this.availabilityCacheCache) {
+      this.availabilityCacheCache = new CollectionWrap(
+        db.collection(AVAILABILITY_CACHE_COLLECTION_NAME),
+      );
+    }
+
+    return this.availabilityCacheCache;
+  }
```

Comment:
<img width="240" alt="image" src="https://github.com/ESP-Number-One/MatchyoSports/assets/67910065/ea1ccc25-91bd-4f30-8a91-8791642750ab">

There are no tests for this, which need to be added